### ISSUE TITLE
Implementatie Stadswarmte NLCS

### ISSUE THREAD

**Issue by Jorn-Wilms-Vattenfall** [nl-digigo/NLCS#477]
Goedemorgen allen.

Hierbij levering deel 1/2 van de NLCS Stadswarmte database. In deze levering zijn de volgende aspecten verwerkt:
1. Layerbenamingen:
- Warmte leidingen nieuw en bestaand;
- Koude leidingen nieuw en bestaand;
- Bronleidingen nieuw en bestaand;
- Warmte componenten nieuw;

2. Nieuwe lijntypes inclusief kleuren:
- Warmte leidingen nieuw en bestaand;
- Koude leidingen nieuw en bestaand;
- Bronleidingen nieuw en bestaand.

Levering 2/2 is nog in ontwikkeling en gaat het volgende bevatten:
- Layerbenamingen warmte vervallen;
- Layerbenamingen warmte componenten bestaand en vervallen;
- Layerbenamingen koude componenten nieuw, bestaand en vervallen;
- Layerbenamingen warmte componenten bestaand en vervallen;
- Blocks van warmte en koude componenten.

Deze issue is aangemaakt in samenspraak met Elisabeth De Vries met als doel feedback te verzamelen over het geleverde.

Met vriendelijke groet,
Jorn Wilms
Vattenfall BU Heat
+31638163027
jorn.wilms@vattenfall.nl

[240823_Vattenfall NLCS Standaard_1.0.pdf](https://github.com/user-attachments/files/16734004/240823_Vattenfall.NLCS.Standaard_1.0.pdf)
[240823_Vattenfall NLCS Standaard_1.0.zip](https://github.com/user-attachments/files/16734003/240823_Vattenfall.NLCS.Standaard_1.0.zip)

**Comment by ElisabethKloren** [comment]
Hier staan de ingevulde importsheeets: https://github.com/nl-digigo/NLCS/tree/main/ontwikkeling/voorstel%205.1/KL/vattenfall%20warmte-%20en%20koude

met deze toelichting: Bijgaand de Template_NLCS_Objecten en de Template_NLCS_Lijntypes ingevuld retour.
Daarnaast ook de lijst met de bestaande objecten (aangeduid in het rood) welke komen te vervallen en worden vervangen door de nieuwe objecten. Ik zag dat er ook een aantal CV objecten tussen staan. Ik heb deze objecten niet aangegeven om te laten vervallen aangezien dit een aparte hoofdgroep is.
De kolom om symbolen toe te wijzen aan de objecten heb ik voor nu nog leeggelaten aangezien de symbolen nog in ontwikkeling zijn.

Ook heb ik het .lin bestand van de nieuwe lijntypes voor de zekerheid toegevoegd, in het geval je deze nog nodig hebt.

**Comment by ElisabethKloren** [comment]
dit is zo netjes voorbereid, dat het meegenomen kan worden in release 5.1

**Comment by ElisabethDeVries** [comment]
@Jorn-Wilms-Vattenfall er stonden nog leestekens in de namen, met name streepjes, die in NLCS een functie hebben voor het herkennen van de onderverdeling van de laagnamen. De definitieve publicatie zal dan ook net andere laagnamen hebben dan jij hebt aangeleverd.

**Comment by Jorn-Wilms-Vattenfall** [comment]
Hi Elisabeth, Dank voor het doorgeven. Dat kan kloppen, helaas worden met de streepjes de DN maten aangeduid met onderscheid tussen de nominale diameter en de buiten diameter. Andere notatie is geen probleem indien de 2 waardes maar niet aan elkaar staan.

**Comment by ElisabethDeVries** [comment]
yes, ik heb spaties tussen de getallen laten staan voor leesbaarheid. Dit komt inderdaad vaker voor, helaas werken we met de beperkingen van wat kan in laagnamen en hoe we in nlcs gebruik maken van streepjes en underscores om posities in de laagnaam weer te geven

**Comment by ElisabethDeVries** [comment]
@Jorn-Wilms-Vattenfall ik heb feedback gekregen van de softwareleveranciers dat voor sommige objecten het "tussenobject" niet aanwezig was, bijvoorbeeld WARMTENET_AANVOER_SPP waarbij er geen WARMTENET_AANVOER was; in dit geval heb ik heb opgelost door er WARMTENET_AANVOER SPP van te maken, bij materialen zoals PE heb ik er een NLCS-Object tussen gezet zodat iemand eventueel een ander materiaal aan kan toevoegen in een eigen bibliotheek.

**Comment by ElisabethDeVries** [comment]
[[Release note]] In de hoofdgroep KL (Kabels en Leidingen) is de indeling van WARMTENETTEN en KOUDENETTEN herzien en uitgebreid met een vrijwillige bijdrage vanuit Vattenfall.
- De aanduiding AANSLUITLEIDING en TRANSPORTLEIDING worden verwijderd uit de harmonicastructuur, er wordt slechts nog onderscheid gemaakt tussen AANVOER, RETOUR en BRONLEIDING. dit leidt tot de verwijdering van een dertigtal laagnamen, en de toevoeging van veel meer laagnamen.
- De gangbare maatvoering van leidingen zijn toegevoegd; van de hulpstukken zijn alleen de soorten opgenomen, de maatvoering kan worden afgeleid uit de aansluitende leidingen. Op deze manier wordt de NLCS-bibliotheek niet te groot.
